owl/SOMA-OBJ.owl

```diff
@@ -1630,6 +1704,28 @@ In robotics, it is not uncommon to encounter shapes that are flipped compared to
     </owl:Class>
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#Storage -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#Storage">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Enclosing"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#affordsTrigger"/>
+                <owl:allValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA-OBJ.owl#StoredObject"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>The disposition of an object (the container) to store other objects by enclosing them to prevent their free movement.</rdfs:comment>
```

Comment:
> 

Does it make sense to say the objects which can be used to store has both dispositions containment and blockage?

Reply:
The same object can be used for multiple objectives and hence can poses many dispositions. But here the comment is leading to the direction of explaining how the container can be used to store things, and by doing so it can indirectly block free movements of things. However, new comment should not lead to such confusion.

Reply:
I will just check with @mpomarlan once today in the meeting. Yes, we can add the disposition as an enabler of common everyday usage. We cant add all the possible dispositions. I did not mean it in that way.